gps_nmea_parser/src/include/gps/gps.h

```diff
@@ -288,7 +380,13 @@ typedef enum {
 } gps_speed_t;
 
 uint8_t     gps_init(gps_t* gh);
+
+#if GPS_CFG_STATUS
+typedef void (*gps_process_cb_t)(gps_statement_t res);
```

Comment:
This should be declared above any function, regardless of configuration setup. It should have description and more importantly, its name should be `gps_process_fn` instead. Function typedefs should always have `xyz_fn` name.

Reply:
ok, see 95d1223